### DISCUSSION TITLE
WCM and WF-Shell needs to be upgrated, please (WayfireWM)

### DISCUSSION THREAD

**Discussion by JamilMGomez** [void-linux/void-packages#50717] [Q&A] [answered]
Hi, since I updated Wayfire to 0.8.1 (which took a long time to get to Void), every time I tried to configure it through WCM (Wayfire Config Manager) it crashes and I got the error 'std::bad_alloc', I thought it was a problem of Wayfire or WCM, but it turns out that someone forgot to update WCM and in the repositories of Void this is still in version 0.7.5, could someone please update it?

Those who use Wayfire daily would be grateful.

PD: Now I noticed that WF-Shell has not been updated either and in a good while! it is still in version 0.7.0 in the repositories and 0.8.1 has been released...

**Comment by pondichys** [comment]
I created a PR #52038 to update wf-shell to 0.8.1 ... in the mean time v 0.9.0 is available but i'll wait a bit for this one :)

**Reply by JamilMGomez** [reply]
Thank you for your time and efford! Is good to know that

At this time I'm using Wayfire 0.8.1, I upgrated via the official method in [https://github.com/WayfireWM/wf-install](Wayfire/wf-install) and is working like a charm

**Comment by ahesford** [comment, marked as answer]
https://github.com/void-linux/void-packages/pull/57076
